mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -77,6 +79,8 @@ CLASS net/minecraft/class_1963 net/minecraft/entity/player/PlayerEntity
 		ARG 3 setSpawn
 	METHOD method_8001 decrementXp (I)V
 		ARG 1 xp
+	METHOD method_8002 canToolBreak (Lnet/minecraft/class_160;)Z
```

Comment:
isUsingEffectiveTool?

Reply:
too long :P

Reply:
`isUsingEffectiveTool` is better
use it